### PR TITLE
Add Netball

### PR DESCRIPTION
**Pull request by danleedham**
I actually built this in 2017, but never got it agreed for use!

### CODE REVIEW

**Comment by danleedham** [comment]
netball

Tested with version 2.1.0 Beta.

**Comment by benjaminkay93** [comment]
are the yellow reg green things meant to do anything on the players?

**Review by benjaminkay93 — APPROVED** [review]

**Comment by benjaminkay93** [comment]
scores all look working, so ive aproved, might be worth removing the player stuff though if its not meant to work :P

File: public/js/dashboard.js [hunk 1]
```diff
@@ -1105,3 +1116,91 @@ app.controller('tennisCGController', ['$scope', 'socket',
...
+                $scope.lancsPlayers.splice(index, 1);
+            }
+        };
+
+        socket.on("netball", function (msg) {
+            $scope.netball = msg;
+        });
+        
+        $scope.quarterChanged = function() {
+            console.log("Quarter");
```
Comment (stephenrob) [inline public/js/dashboard.js:1185]: Do we need to console log here?

**Review by stephenrob — COMMENTED** [review]
Stray console logs, Looks like it should be fine besides. To help with readability in future full phrases like "currentQuarter" is better than "currQuarter".

File: public/js/dashboard.js (hunk 1 above)
Comment (moschopsuk) [reply public/js/dashboard.js:1185]: 👀

**Pull request merged**